Mailnag/backends/__init__.py

```diff
@@ -41,7 +41,7 @@ def _str_to_folders(folders_str):
 
 
 def _folders_to_str(folders):
-	return json.dumps(folders)
+	return json.dumps(folders, ensure_ascii=False).encode('utf-8')
```

Comment:
It could be moved encode from Maidir to loads and Maildir would be happy with that, but then imap backend fails, because in commit https://github.com/pulb/mailnag/pull/140/commits/08e356ac685172b16f50d624ddf1e18d8be83cc6 I moved encode_mutf7 call to imap backend and encode_mutf7 requires unicode parameter. I don't want to modify encode_mutf7.